app/merchant/forms.py

```diff
@@ -0,0 +1,9 @@
+from flask.ext.wtf import Form
+from wtforms.fields import IntegerField
+from wtforms.validators import InputRequired
+
+
+class CartQuantityForm(Form):
+    quantity = IntegerField('Quantity', validators=[
+        InputRequired(),
+    ])
```

Comment:
I deleted this form from master recently; it's not being used anywhere so we can get rid of it.